x/auth/vesting/client/cli/tx.go

```diff
@@ -261,3 +263,50 @@ func NewMsgClawbackCmd() *cobra.Command {
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+// NewMsgCreateDelayedVestingAccountCmd returns a CLI command handler for creating a
+// NewMsgCreateDelayedVestingAccountCmd transaction.
+// This is hacky, but meant to mitigate the pain of a very specific use case.
+// Namely, make it easy to make cliff locks to an address.
+func NewMsgCreateCliffVestingAccountCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-cliff-vesting-account [to_address] [amount] [cliff_duration]",
+		Short: "Create a new cliff vesting account funded with an allocation of tokens.",
+		Long: `Create a new delayed vesting account funded with an allocation of tokens. All vesting accouts created will have their start time
+set by the committed block's time. The cliff duration should be specified in hours.`,
+		Args: cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			toAddr, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
+				return err
+			}
+
+			amount, err := sdk.ParseCoinsNormalized(args[1])
+			if err != nil {
+				return err
+			}
+
+			cliffDuration, err := time.ParseDuration(args[2])
+			if err != nil {
+				fmt.Println("duration incorrectly formatted, see https://pkg.go.dev/time#ParseDuration")
+				return err
+			}
+			cliffVesting := true
```

Comment:
Suggest making this as constant at the top of the file for readability and to avoid initializing it every time the command is called

Reply:
eh, I think the context is weird given a different API for clawback work